### DISCUSSION TITLE
I am a schizophrenic new to

### DISCUSSION THREAD

**Discussion by CrocTomexogon** [community/community#131056] [Accessibility]
### Select Topic Area

Question

### GitHub Feature Area

Actions

### Body

Hi

I am a bipolar schizophrenic please assist me on how to use this platform

**Comment by Akash1134** [comment]
Hi @CrocTomexogon ,

GitHub's user interface can adapt to your vision, hearing, motor, cognitive, or learning needs. Please explore [this post](https://accessibility.github.com/) to learn more.

Thanks!

**Reply by nl8-gh** [reply]
How does the shiny marketing prose you referred to actually explain **any** relevant setting (or where to find them) to adjust GH for e.g. people with disabilities, low vision, … ?

**Comment by OCEANOFANYTHINGOFFICIAL** [comment]
Hi there @CrocTomexogon ,

First of all, I want to commend you for reaching out. Using a platform like GitHub can be a valuable way to engage with the tech community, share your projects, and contribute to open source. Here are some steps and resources to help you get started:

1. **Creating an Account**:
   - Go to [GitHub's sign-up page](https://github.com/join) and create an account by following the instructions provided.

2. **Getting Familiar with GitHub**:
   - **GitHub Guides**: Start with the [GitHub Guides](https://guides.github.com/). They cover everything from the basics to more advanced features.
   - **Hello World Tutorial**: GitHub has a [Hello World](https://guides.github.com/activities/hello-world/) tutorial that is perfect for beginners.

3. **Using GitHub**:
   - **Repositories**: Learn how to create a repository, which is where your project's files and history will be stored. Check out the [creating a repository](https://docs.github.com/en/repositories/creating-and-managing-repositories/creating-a-new-repository) guide.
   - **Commits**: Understand how to save changes with commits. The [about commits](https://docs.github.com/en/commits) page can help.
   - **Issues**: You can track your tasks or report bugs using GitHub Issues. Here's more on [managing issues](https://docs.github.com/en/issues/tracking-your-work-with-issues).

4. **Community and Support**:
   - **GitHub Community Forum**: Join the [GitHub Community Forum](https://github.community/) where you can ask questions and get help from other users.
   - **GitHub Support**: If you need more direct assistance, GitHub Support can help. Visit their [support page](https://support.github.com/contact) to reach out.

5. **Mental Health Resources**:
   - Using GitHub should be an enjoyable experience, but it’s important to take care of your mental health. The [National Alliance on Mental Illness (NAMI)](https://www.nami.org/Home) offers resources and support for managing mental health conditions.

If you have any specific questions or need further assistance, feel free to ask here or in the GitHub Community Forum. We’re here to help!

Take care, and best of luck with your coding journey!
